content/zh/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -1,180 +1,829 @@
 
-一般来说，Pod 不会消失，直到人为销毁他们。这可能是一个人或控制器。这个规则的唯一例外是成功或失败的 `phase` 超过一段时间（由 master 确定）的Pod将过期并被自动销毁。
+If a container has a `preStop` hook configured, that runs before the container enters
+the `Terminated` state.
+-->
+### `Terminated`（已终止）   {#container-state-terminated}
 
-有三种可用的控制器：
+处于 `Terminated` 状态的容器已经开始执行并且或者正常结束或者因为某些原因失败。
+如果你使用 `kubectl` 来查询包含 `Terminated` 状态的容器的 Pod 时，你会看到
+容器进入此状态的原因、退出代码以及容器执行期间的起止时间。
 
-- 使用 [Job](/docs/concepts/jobs/run-to-completion-finite-workloads/) 运行预期会终止的 Pod，例如批量计算。Job 仅适用于重启策略为 `OnFailure` 或 `Never` 的 Pod。
+如果容器配置了 `preStop` 回调，则该回调会在容器进入 `Terminated`
+状态之前执行。
 
+<!--
+## Pod conditions
 
-- 对预期不会终止的 Pod 使用 [ReplicationController](/docs/concepts/workloads/controllers/replicationcontroller/)、[ReplicaSet](/docs/concepts/workloads/controllers/replicaset/) 和 [Deployment](/docs/concepts/workloads/controllers/deployment/) ，例如 Web 服务器。 ReplicationController 仅适用于具有 `restartPolicy` 为 Always 的 Pod。
-- 提供特定于机器的系统服务，使用 [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) 为每台机器运行一个 Pod 。
+A Pod has a PodStatus, which has an array of
+[PodConditions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core)
+through which the Pod has or has not passed:
+-->
+## Pod 状况  {#pod-conditions}
 
-所有这三种类型的控制器都包含一个 PodTemplate。建议创建适当的控制器，让它们来创建 Pod，而不是直接自己创建 Pod。这是因为单独的 Pod 在机器故障的情况下没有办法自动复原，而控制器却可以。
+Pod 有一个 PodStatus 对象，其中包含一个
+[PodConditions](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#podcondition-v1-core)
+数组。Pod 可能通过也可能未通过其中的一些状况测试。
 
-如果节点死亡或与集群的其余部分断开连接，则 Kubernetes 将应用一个策略将丢失节点上的所有 Pod 的 `phase` 设置为 Failed。
+<!--
+* `PodScheduled`: the Pod has been scheduled to a node.
+* `ContainersReady`: all containers in the Pod are ready.
+* `Initialized`: all [init containers](/docs/concepts/workloads/pods/init-containers/)
+  have started successfully.
+* `Ready`: the Pod is able to serve requests and should be added to the load
+  balancing pools of all matching Services.
+-->
+* `PodScheduled`：Pod 已经被调度到某节点；
+* `ContainersReady`：Pod 中所有容器都已就绪；
+* `Initialized`：所有的 [Init 容器](/zh/docs/concepts/workloads/pods/init-containers/)
+  都已成功启动；
+* `Ready`：Pod 可以为请求提供服务，并且应该被添加到对应服务的负载均衡池中。
 
-## 示例
+<!--
+Field name           | Description
+`type`               | Name of this Pod condition.
+`status`             | Indicates whether that condition is applicable, with possible values "`True`", "`False`", or "`Unknown`".
+`lastProbeTime`      | Timestamp of when the Pod condition was last probed.
+`lastTransitionTime` | Timestamp for when the Pod last transitioned from one status to another.
+`reason`             | Machine-readable, UpperCamelCase text indicating the reason for the condition's last transition.
+`message`            | Human-readable message indicating details about the last status transition.
+-->
+字段名称             | 描述
+:--------------------|:-----------
+`type`               | Pod 状况的名称
+`status`             | 表明该状况是否适用，可能的取值有 "`True`", "`False`", or "`Unknown`"
```

Comment:
I'd expect to see “`True`, `False` or `Unknown`” translated here - it's not the logical OR operation.

Reply:
True, False, Unknown are the verbal values shown in `kubectl` output. It is not supposed to be translated.
However, the 'or' should be translated.
Fixed.